Jenkinsfile

```diff
@@ -128,6 +132,9 @@ node('ubuntu-chef-zion') {
             git commit -m '${commitMessage}'
             git push https://${env.GITHUB_API_USERNAME}:${env.GITHUB_API_PASSWORD}@github.com/${organization}/${repository}.git ${branch}
           """)
+
+          version = versionTools.getCommitVersion(majorMinorVersion, OsTools.runSafe(this, "git rev-parse HEAD"))
```

Comment:
I don't use OsTools any more. what is the point of it?
sh "git rev-parse HEAD"
is far simpler and better.